### ISSUE TITLE
Sidekiq enterprise and Sidekiq::Limiter::OverLimit

### ISSUE THREAD

**Issue by Piioo** [fastly/sidekiq-prometheus#41]
Hello,

We discovered errors on our system after the update to version 2.0.0 in connection with sidekiq. (1.9.0 too)

We do not have the sidekiq enterprise tool, so our application doesn't know this class `Sidekiq::Limiter::OverLimit`

For this reason, this line generates errors for us:
https://github.com/fastly/sidekiq-prometheus/blob/main/lib/sidekiq_prometheus/job_metrics.rb#L33

```
NameError: uninitialized constant Sidekiq::Limiter (NameError)

      if e.instance_of?(::Sidekiq::Limiter::OverLimit)
                                          ^^^^^^^^^^^
  from sidekiq_prometheus (2.0.0) lib/sidekiq_prometheus/job_metrics.rb:33:in `rescue in call'
  from sidekiq_prometheus (2.0.0) lib/sidekiq_prometheus/job_metrics.rb:32:in `call'
  from sidekiq (7.1.2) lib/sidekiq/middleware/chain.rb:182:in `traverse'
  from sidekiq (7.1.2) lib/sidekiq/middleware/chain.rb:183:in `block in traverse'
  from prometheus_exporter (2.0.8) lib/prometheus_exporter/instrumentation/sidekiq.rb:53:in `call'
  from sidekiq (7.1.2) lib/sidekiq/middleware/chain.rb:182:in `traverse'
  from sidekiq (7.1.2) lib/sidekiq/middleware/chain.rb:183:in `block in traverse'
  from sentry-sidekiq (5.10.0) lib/sentry/sidekiq/sentry_context_middleware.rb:26:in `call'
  from sidekiq (7.1.2) lib/sidekiq/middleware/chain.rb:182:in `traverse'
  from sidekiq (7.1.2) lib/sidekiq/middleware/chain.rb:183:in `block in traverse'
  from sidekiq (7.1.2) lib/sidekiq/metrics/tracking.rb:26:in `track'
  from sidekiq (7.1.2) lib/sidekiq/metrics/tracking.rb:122:in `call'
  from sidekiq (7.1.2) lib/sidekiq/middleware/chain.rb:182:in `traverse'
  from sidekiq (7.1.2) lib/sidekiq/middleware/chain.rb:173:in `invoke'
  from sidekiq (7.1.2) lib/sidekiq/processor.rb:176:in `block in process'
  from sidekiq (7.1.2) lib/sidekiq/processor.rb:140:in `block (6 levels) in dispatch'
  from sidekiq (7.1.2) lib/sidekiq/job_retry.rb:114:in `local'
  from sidekiq (7.1.2) lib/sidekiq/processor.rb:139:in `block (5 levels) in dispatch'
  from sidekiq (7.1.2) lib/sidekiq/rails.rb:16:in `block in call'
  from activesupport (7.0.7.2) lib/active_support/execution_wrapper.rb:92:in `wrap'
  from activesupport (7.0.7.2) lib/active_support/reloader.rb:72:in `block in wrap'
  from activesupport (7.0.7.2) lib/active_support/execution_wrapper.rb:92:in `wrap'
  from activesupport (7.0.7.2) lib/active_support/reloader.rb:71:in `wrap'
  from sidekiq (7.1.2) lib/sidekiq/rails.rb:15:in `call'
  from sidekiq (7.1.2) lib/sidekiq/processor.rb:135:in `block (4 levels) in dispatch'
  from sidekiq (7.1.2) lib/sidekiq/processor.rb:270:in `stats'
  from sidekiq (7.1.2) lib/sidekiq/processor.rb:130:in `block (3 levels) in dispatch'
  from sidekiq (7.1.2) lib/sidekiq/job_logger.rb:13:in `call'
  from sidekiq (7.1.2) lib/sidekiq/processor.rb:129:in `block (2 levels) in dispatch'
  from sidekiq (7.1.2) lib/sidekiq/job_retry.rb:81:in `global'
  from sidekiq (7.1.2) lib/sidekiq/processor.rb:128:in `block in dispatch'
  from sidekiq (7.1.2) lib/sidekiq/job_logger.rb:39:in `prepare'
  from sidekiq (7.1.2) lib/sidekiq/processor.rb:127:in `dispatch'
  from sidekiq (7.1.2) lib/sidekiq/processor.rb:175:in `process'
  from sidekiq (7.1.2) lib/sidekiq/processor.rb:82:in `process_one'
  from sidekiq (7.1.2) lib/sidekiq/processor.rb:72:in `run'
  from sidekiq (7.1.2) lib/sidekiq/component.rb:10:in `watchdog'
  from sidekiq (7.1.2) lib/sidekiq/component.rb:19:in `block in safe_thread'
```

**Comment by jayroh** [comment]
Yes. Ran into the same thing here.

**Comment by leklund** [comment]
Fixed in #42 

v2.0.1 released
